api.py

```diff
@@ -223,6 +223,39 @@ def _get_all_pods():
         pods.append(i.metadata.name)
     return pods
 
+# work in progress for log stream endpoint
+@api.route("/stream_logs")
+def stream_logs_html(req, resp):
+    """Route to stream logs"""
+    resp.content = api.template(
+        'stream_logs.html',
+        log_stream=_get_log_stream()
```

Comment:
You'll want to pass in a few parameters here, name and namespace at least

Reply:
I added a commit.